### DISCUSSION TITLE
Livewire component rendering the view for another component

### DISCUSSION THREAD

**Discussion by theJohnCode** [livewire/livewire#5631] [1. Help]
I ran into this weird issue, I created a component PaymentComponent,
 after I am done with what I am doing there, I created another one called PaymentManage.
 I am using full page rendering, now instead the PaymentManage class will use its own view,
 it is rendering the view for PaymentComponent. 
I have deleted the new component and recreated it but it is still down the same thing.

Have you anyone encountered such an error?

**Comment by joshhanley** [comment]
@theJohnCode can you share your components classes and views so we can have a look?
